src/Orc.Extensibility/Services/PluginFinderBase.cs

```diff
@@ -346,15 +348,19 @@ protected virtual async Task FindPluginsInAssemblyAsync(PluginProbingContext con
                 return;
             }
 
+            // Just use file name as checksum, it will make it unique as well
+            var fileRuntimeAssembly = new FileRuntimeAssembly(assemblyPath, assemblyPath, assemblyPath, assemblyPath);
```

Comment:
is it worth to create a ctor with single parameter for FileRuntimeAssembly?

Reply:
have considered, will add